amp-conformance/run_tests.pl.in

```diff
@@ -30,12 +32,13 @@ use File::Basename;
 use File::Find;
 use Getopt::Long;
 use Safe;
+use strict;
 use threads;
 use threads::shared;
-use strict;
+use warnings;
 
 my $max_num_threads = @AMP_CONFORMANCE_NUM_THREADS@;
-my $use_threads = $Config{useithreads} and $max_num_threads > 1;
+my $use_threads = $Config{useithreads};# && $max_num_threads > 1; - temporarily disabled.
```

Comment:
That was / is part of a separate experiment, since there's a wee issue with how the Perl is laid out right now, however it shouldn't be part of this particular batch of changes, I reverted it. Thank you for pointing it out.